spec/unit/rom/elasticsearch/relation/dataset_spec.rb

```diff
@@ -34,15 +34,17 @@
         end
 
         schema do
-          attribute :id, ROM::Types::Int
+          attribute :id, ROM::Types::Integer
           attribute :name, ROM::Types::String
           attribute :_metadata, ROM::Types::Hash,
-                    read: ROM::Types::Hash.symbolized(_index: 'string')
+                    read: ROM::Types::Hash.schema(_index: ROM::Types::String)
+                                          .with_key_transform(&:to_sym)
         end
       end
 
       users.create_index
       users.command(:create).call(id: 1, name: 'Jane')
+      users.refresh
```

Comment:
Why was this added?

Reply:
`create` command is asynchronous, that's why. I assume it just was missed. I ran tests before commit and came across this.